faker/providers/date_time/__init__.py

```diff
@@ -20,6 +21,8 @@
 
 def datetime_to_timestamp(dt):
     if getattr(dt, 'tzinfo', None) is not None:
+        if getattr(dt, 'tzinfo', 'UTC'):
```

Comment:
we already checked that `dt.tzinfo` exists and is not None.

The only way L25 will not be executed is if `dt.tzinfo` is either `False` or `0`, which I is never the case, meaning L25 will always be executed and this `if` is unnecessary